data/mods/ssb/random-teams.js

```diff
@@ -402,6 +402,12 @@ class RandomStaffBrosTeams extends RandomTeams {
 				signatureMove: 'Lilypad Shield',
 				evs: {hp: 252, def: 60, spd: 196}, ivs: {atk: 0}, nature: 'Calm',
 			},
+			'pre': {
+				species: 'Deoxys', ability: 'Optimize', item: 'Rocky Helmet', gender: '',
```

Comment:
`'N'` is used for none, a blank is random gender.

Reply:
Thanks.